app/components/Navbar.tsx

```diff
@@ -61,10 +62,16 @@ export const Navbar = () => {
             <div className="relative">
               <Button
                 variant="pastel"
-                className="space-x-1 text-nowrap"
+                className="space-x-1.5 text-nowrap"
                 onClick={() => trackEvent(FA_EVENTS.BETS.MY_BETS)}
               >
-                My bets
+                <Image
+                  src="assets/icons/bets.svg"
+                  width={16}
+                  height={16}
+                  alt="my bets icon"
+                />
+                <p> My bets</p>
               </Button>
               {hasUnredeemedBets && (
                 <div className="absolute -right-2 -top-1 flex size-5 items-center justify-center rounded-100 border-2 border-surface-surface-bg bg-surface-success-main p-1 text-text-white">
```

Comment:
Can we fix the text color has well?

Reply:
what's wrong with the text color?

Reply:
low visibility on dark mode

Reply:
something like text-text-neutral-alt-white

Reply:
I think you meant the icon color, as the text is the same as it was (an the others), I'll take a look. 
The idea was to have a lit bit less contrast.

Reply:
I'm suggesting to change the unredeem bets bubble text color, it works on light mode but in dark mode is not easy to read. The green brightness merges with the text white color, creating less contrast with the background, so it becomes difficult to read in dark mode.